cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicas.java

```diff
@@ -43,12 +43,13 @@
  *
  * <p>
  *   The SortedReplicas are initialized lazily, i.e. until one of {@link #sortedReplicas()},
- *   {@link #reverselySortedReplicas()} and {@link #reverselySortedReplicaWrappers()} is invoked, the sorted replicas
+ *   {@link #reverselySortedReplicas()} and {@link #sortedReplicaWrappers()} is invoked, the sorted replicas
  *   will not be populated.
  * </p>
  */
 public class SortedReplicas {
   private final Broker _broker;
+  private final Disk _disk;
```

Comment:
Not necessary, so for this class RAID broker will just be a special case of JBOD broker(which only has 1 disk), you can still get a `SortedReplicas` object for that RAID disk.

Actually we allow mix of JBOD and non-JBOD, if we run rebalance_disk on such cluster, for RAID broker, `IntraBrokerDiskCapacityGoal` will act like `DiskCapacityGoal`( if the RAID disk is overloaded, OptimizationException will be thrown) and `IntraBrokerDiskUsageDistributionGoal` will be a no-op(since disk utilization percentage will equal to broker utilization percentage and will not be treated as unbalanced).